github/private_registries.go

```diff
@@ -0,0 +1,240 @@
+//
+//meta:operation POST /orgs/{org}/private-registries
+func (s *PrivateRegistriesService) CreateOrganizationPrivateRegistries(ctx context.Context, org string, privateRegistry CreateOrganizationPrivateRegistry) (*PrivateRegistry, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/private-registries", org)
+
+	req, err := s.client.NewRequest("POST", u, privateRegistry)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var result PrivateRegistry
+	resp, err := s.client.Do(ctx, req, &result)
+	if err != nil {
+		return nil, resp, err
+	}
+	return &result, resp, nil
+}
+
+// GetOrganizationPrivateRegistriesPublicKey retrieves the public key for encrypting secrets for an organization's private registries.
+//
+// GitHub API docs: https://docs.github.com/rest/private-registries/organization-configurations#get-private-registries-public-key-for-an-organization
+//
+//meta:operation GET /orgs/{org}/private-registries/public-key
+func (s *PrivateRegistriesService) GetOrganizationPrivateRegistriesPublicKey(ctx context.Context, org string) (*PublicKey, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/private-registries/public-key", org)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var publicKey PublicKey
+	resp, err := s.client.Do(ctx, req, &publicKey)
+	if err != nil {
+		return nil, resp, err
+	}
+	return &publicKey, resp, nil
+}
+
+// GetOrganizationPrivateRegistry gets a specific private registry for an organization.
+// name parameter is the name of the private registry to delete, PrivateRegistry.Name
+//
+// GitHub API docs: https://docs.github.com/rest/private-registries/organization-configurations#get-a-private-registry-for-an-organization
+//
+//meta:operation GET /orgs/{org}/private-registries/{secret_name}
+func (s *PrivateRegistriesService) GetOrganizationPrivateRegistry(ctx context.Context, org, name string) (*PrivateRegistry, *Response, error) {
+	u := fmt.Sprintf("orgs/%v/private-registries/%v", org, name)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var privateRegistry PrivateRegistry
+	resp, err := s.client.Do(ctx, req, &privateRegistry)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return &privateRegistry, resp, nil
+}
+
+// UpdateOrganizationPrivateRegistry updates a specific private registry for an organization.
+// name parameter is the name of the private registry to delete, PrivateRegistry.Name
+//
+// GitHub API docs: https://docs.github.com/rest/private-registries/organization-configurations#update-a-private-registry-for-an-organization
+//
+//meta:operation PATCH /orgs/{org}/private-registries/{secret_name}
+func (s *PrivateRegistriesService) UpdateOrganizationPrivateRegistry(ctx context.Context, org, name string, privateRegistry UpdateOrganizationPrivateRegistry) (*PrivateRegistry, *Response, error) {
```

Comment:
```suggestion
func (s *PrivateRegistriesService) UpdateOrganizationPrivateRegistry(ctx context.Context, org, secretName string, privateRegistry UpdateOrganizationPrivateRegistry) (*PrivateRegistry, *Response, error) {
```

<img width="565" height="447" alt="Image" src="https://github.com/user-attachments/assets/9b67be2f-f379-40d2-b9c2-dd53659ef23c" />